cli/service/service.go

```diff
@@ -76,6 +81,59 @@ func List() {
 	fmt.Printf("%s\n", jsonBytes)
 }
 
+func Log(ServiceName string) {
+	msgPrinter := i18n.GetMessagePrinter()
+	for {
+		// Open syslog
+		file, err := os.Open("/var/log/syslog")
+		if err != nil {
+			cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("/var/log/syslog could not be opened or does not exist", err))
+		}
+		// Check file stats - size of file
+		fi, err := file.Stat()
+		if err != nil {
+			cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("/var/log/syslog could not get stats", err))
+		}
+		file_size := fi.Size()
+		defer file.Close()
+		reader := bufio.NewReader(file)
+		for {
+			// open file again to re-check stats in case it was logrotated
+			new_file, err := os.Open("/var/log/syslog")
+			if err != nil {
+				continue
```

Comment:
Why continue? If you want to retry the open because of log rotate then you should probably also sleep for a second.  Same question on the Stat call below.